mne/label.py

```diff
@@ -1925,6 +1930,20 @@ def _get_annot_fname(annot_fname, subject, hemi, parc, subjects_dir):
     return annot_fname, hemis
 
 
+def _load_vert_pos(subject, subjects_dir, surf_name, hemi, n_expected,
+                   extra=''):
+    fname_surf = op.join(subjects_dir, subject, 'surf',
+                         '%s.%s' % (hemi, surf_name))
+    vert_pos, _ = read_surface(fname_surf)
+    vert_pos /= 1e3  # the positions in labels are in meters
```

Comment:
`vert_pos = read_surface(fname_surf)[0] / 1e3  # cm to meters`

Reply:
Actually @agramfort prefers it as `a, _ = ` to be explicit about arguments and backward compat, etc.